geps/gep-2648/index.md

```diff
@@ -132,7 +129,35 @@ when a Policy is a Direct Attached Policy:
   of some sort, where the things outside the namespace can opt–out of the behavior.
   (Notably, this is the design that we used for ReferenceGrant).
 
-## Apply Policies to Sections of a Resource
+## Target References
+### Cross Namespace
+
+In all cases, Gateway API policies should only have an effect on the namespace
+they exist within. In the case of policies that could apply to mesh
+implementations, it may be desirable to have a policy that affects traffic
+originating from the local namespace but going to a separate namespace. Unless
+that specific case is desired, all policy target refs should be local and
+exclude the "namespace" field.
+
+### Multiple
+
+In some cases, it may be desirable for a policy to target more than one resource
```

Comment:
@robscott have you considered allowing multiple targetRefs ONLY for inherited policies, and disallowing in direct attach policies?  I thought the spirit of direct attach policies was that they were one-to-one and the ancestor status was clear.

Reply:
I personally see value in supporting multiple targetRefs at both, Direct and Inherited policies. Rationale:
- One who targets multiple HTTPRoutes with a Direct policy can potentially manage only one policy object instead of also many, less repetition, etc.
- Simplicity, the only difference between the two would continue to be, as of today, the `sectionName`.

Especially the latter I believe to be consistent in the long run with the possibility of Direct vs Inherited being reduced to different _merge strategies_, and not necessarily two big classes of policies – see https://github.com/kubernetes-sigs/gateway-api/discussions/2927#discussion-6453213.

Reply:
I just added a [separate response](https://github.com/kubernetes-sigs/gateway-api/pull/2966#discussion_r1566441322) that I think is related to this question too. Essentially I agree that we're headed in that direction with #2927, but I'm trying to focus this on the smallest set of changes required to enable changes to our Backend policies in v1.1, which both happen to be direct. I think once someone has time to formally propose the rest of the changes discussed in #2927 and make the corresponding updates to the GEP, all of this will apply equally to direct and inherited policies.